src/MusicStore/Controllers/AccountController.cs

```diff
@@ -65,6 +71,7 @@ public IActionResult Login(string returnUrl = null)
             }
             else
             {
+                _logger.LogInformation("Failed to login {userName}.", model.Email);
```

Comment:
Plug, `login` is a noun, but `log in` is a verb phrase, and I think you want the verb.